index.js

```diff
@@ -5,9 +5,7 @@ humburger.onclick = function () {
 };
 
 const xmark = document.querySelectorAll('.exam');
-for (let i = 0; i < xmark.length; i += 1) {
-  xmark[i].onclick = function () {
-    const navBar = document.querySelector('.menue-btn');
-    navBar.classList.remove('active');
+xmark.onclick = function () {
```

Comment:
- [ ] I noticed you made some changes that affected the mobile menu close bar, it was working when the previous reviewer did review it but at the moment, it is not working, definitely you made changes that affected it, and I would implore you to dig further and fix it, other than this, you did a good job.
![image](https://user-images.githubusercontent.com/86200356/231456597-f3a143df-4aa2-46e3-9046-49e53296d30f.png)